### DISCUSSION TITLE
what ubuntu version supported?

### DISCUSSION THREAD

**Discussion by Necrosis000** [neutrinolabs/xrdp#3217] [Q&A] [answered]
just a quick question but what is the currently supported ubuntu version?

**Comment by matt335672** [comment]
What do you mean @Necrosis000? Do you want to install it or compile it?

If the former, use `apt install` to get the version built for your Ubuntu version from the repositories.

**Reply by Necrosis000** [reply, marked as answer]
my apologies, I had a problem where i would only see a blue screen but i just had to downgrade my ubuntu from 24 to 22 and it fixed it
